components/com_users/tmpl/profile/default_custom.php

```diff
@@ -43,7 +42,13 @@
             <?php endif; ?>
             <dl class="dl-horizontal">
                 <?php foreach ($fields as $field) : ?>
-                    <?php if (!$field->hidden && $field->type !== 'Spacer') : ?>
+                    <?php if (!$field->hidden && $field->type !== 'Spacer') :
+                        if($field->type === 'Subform' && $field->fieldname === 'row') :
+                            // Correct the fieldname so that subform custom fields show up.
+                            preg_match("/jform\[com_fields]\[(.*)]/",$field->name,$matches);
```

Comment:
```suggestion
                        if ($field->type === 'Subform' && $field->fieldname === 'row') :
                            // Correct the fieldname so that subform custom fields show up.
                            preg_match("/jform\[com_fields]\[(.*)]/", $field->name, $matches);
```
Fix code style errors reported by drone.